.github/workflows/ci-cd-pims-dev.yml

```diff
@@ -26,13 +26,10 @@ env:
   TAG_PROD: "prod"
 
 on:
-  pull_request_target:
-    branches: [dev]
-    types: [closed]
```

Comment:
I like this change but it will lead to a change in our process. From now on, developers will be responsible to trigger the CI build for DEV environment themselves, right? 
We should talk about how would that look like within the team